src/librustc_driver/driver.rs

```diff
@@ -952,15 +901,16 @@ pub fn default_provide(providers: &mut ty::maps::Providers) {
     lint::provide(providers);
 }
 
-pub fn default_provide_extern(providers: &mut ty::maps::Providers) {
+pub fn default_provide_extern(trans: &TransCrate, providers: &mut ty::maps::Providers) {
     cstore::provide_extern(providers);
-    DefaultTransCrate::provide_extern(providers);
+    trans.provide_extern(providers);
```

Comment:
Can you call this (and the one in `default_provide`) in `phase_3_run_analysis_passes`, after the calls to `default_provide` and `default_provide_extern`, respectively?
These two functions are intended to be the minimum necessary to run analysis and should not assume a backend.

Reply:
Done locally